en/application.texy

```diff
@@ -83,11 +83,11 @@ Nette Application
 
 The Application class has a single task: to respond to an HTTP request.
 
-Applications written in Nette are divided into many so-called presenters (in other frameworks the term controller is used), which are classes representing a specific website page: eg homepage; product in e-shop; sign-in form; sitemap feed, etc. The application can have from one to thousands of presenters.
+Applications written in Nette are divided into many so-called presenters (in other frameworks known as controllers), which are classes representing a specific website page: eg homepage; product in e-shop; sign-in form; sitemap feed, etc. The application can have from one to thousands of presenters.
 
-The application starts by asking the so-called router to decide which of the presenters to pass the current request for processing. The router decides whose responsibility it is. He looks at the input URL `https://example.com/product/123` and, based on how it is set up, decides that this is a job, for example, for **presenter** `Product`,  who wants to `show` a product with `id=123` as an action. It is a good habit to write a pair of presenter + action separated by a colon as `Product:show`.
+The application starts by asking the so-called router to decide which of the presenters to pass the current request for processing. The router decides whose responsibility it is. He looks at the input URL `https://example.com/product/123` and, based on how he is set up, decides that this is a job, for example, for **presenter** `Product`,  who wants to `show` a product with `id=123` as an action. It is a good habit to write a pairs of presenter + action separated by a colon as `Product:show`.
```

Comment:
Tady by spíš mělo být `It looks` a `how it is set up`, ne?

Reply:
Neni problem, hlavne by to bylo jednotne a ne He/it  :) opravim

Reply:
A dá se vůbec použít he, pokud se nebavíme o lidech? Nevyznám se v tom…

Reply:
Personifikace podle je porad validni zpusob i v anglictine. Zalezi na tom jakou formou tu docku chces vlastne vest, vetsinou se asi pouziva It